cloudhub-2/modules/ROOT/pages/_partials/create-config.adoc

```diff
@@ -33,7 +33,7 @@ MuleSoft recommends using subnets from standard RFC 1918 addresses, such as `10.
 
 Optionally, if you want to connect your private space to your corporate network (either on-premises or in the cloud), specify reserved CIDR blocks for your private space to prevent IP address overlap. Use CIDR notation and commas: `192.168.0.0/16, 172.16.0.0/12`.
 
-You cannot use the following reserved CIDR blocks:
+You cannot use the following CIDR blocks:
```

Comment:
the list below is specifically unusable for PS network CIDR. but the last 4 can be used as part of reserved corporate CIDRs. that's why we need to separate them.